File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_4/exercice2.py
T=[None for i in range(3)]
for i in range(0,3):
    T[i]=int(input(f"saisier l'entier numero {i+1}: "))
min=T[0]
for i in range(0,3):
    if T[i]<=min :
        min=T[i]
        min_pos=i+1
print("le plus petit élément de ce tableau est : ",min)
print("et sa position est ",min_pos)

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_4/exercice2.c
#include <stdio.h>
int min,min_pos,T[30];
int main(){
    for(int i=0;i<30;i++){
        printf("saisir l'entier numero %d: ",i+1);
        scanf("%d",&T[i]);
    }
    min=T[0];
    for(int i=0;i<30;i++){
        if(T[i]<=min){
            min=T[i];
            min_pos=i+1;
        }
    }
    printf("le plus petit élément de ce tableau est : %d\n",min);
    printf("et sa position est %d\n",min_pos);
    return 0;
}
